community-operators/rook/rook.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,273 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "ceph.rook.io/v1",
```

Comment:
Looks like you have 7 CRDs but only an example for the `CephCluster` CR. It would be nice if all CRDs would come with an example.